src/atap_widgets/concordance.py

```diff
@@ -134,10 +143,147 @@ def prepare_text_df(
 
     if isinstance(language_model, str):
         language_model = spacy.load(language_model)
-    output["spacy_doc"] = output["text"].map(language_model)
+    output["spacy_doc"] = output["text"].map(language_model) #Doc for each line
+    
     return output
 
+class DataIngest():
```

Comment:
I think ideally names for functions/classes should mention what they achieve - maybe `ConcordanceLoader` would be a better name for this?

Reply:
done